app/src/androidTestServer/java/io/appium/uiautomator2/server/test/AppiumUiAutomator2Server.java

```diff
@@ -35,11 +43,47 @@ public class AppiumUiAutomator2Server {
     @Test
     public void startServer() throws InterruptedException {
         Logger.info( ">>>Entry point start<<<");
+        Logger.info(buildConfigSerializer());
         ServerInstrumentation.getInstance().start();
         CountDownLatch shutdownLatch = ServerInstrumentation.getInstance().getShutdownLatch();
         if (shutdownLatch != null) {
             shutdownLatch.await();
         }
         Logger.info(">>>Entry point finish<<<");
     }
+    /**
+     * Serializes all static fields of the BuildConfig class into a JSON string.
+     *
+     * <p>This method uses reflection to collect static field values from the BuildConfig class
+     * and converts them into a JSON format using Gson library.</p>
+     *
+     * @return JSON string representation of BuildConfig's static fields
+     * @throws RuntimeException if BuildConfig class cannot be found or field access is denied
+     *
+     * @apiNote The BuildConfig class name is hardcoded to "io.appium.uiautomator2.test.BuildConfig".
+     *          Modify the class name if used in different packages.
+     */
+    public static String buildConfigSerializer() {
+        final String buildConfigClassName = "io.appium.uiautomator2.test.BuildConfig";
+        final Map<String, Object> fieldMap = new HashMap<>();
+
+        try {
+            Class<?> buildConfigClass = Class.forName(buildConfigClassName);
+
+            for (Field field : buildConfigClass.getDeclaredFields()) {
+                if (Modifier.isStatic(field.getModifiers())) {
+                    try {
+                        field.setAccessible(true);
+                        Object value = field.get(null);
+                        fieldMap.put(field.getName(), value);
+                    } catch (IllegalAccessException e) {
+                        throw new RuntimeException("Field access denied for: " + field.getName(), e);
```

Comment:
I assume this method call should be safe as it is only used for logging. All exceptions could simply be dumped to logs, but it should not fail the server execution